### PR TITLE
[DMARC Management] Remove manage sources

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-7149

### CODE REVIEW

File: content/dmarc-management/statistics.md [hunk 1]
```diff
@@ -14,4 +14,16 @@ DMARC Management (beta) allows you to review how emails sent on your behalf have
...
 4. Moving your mouse through the graph gives you details for a particular day. Select **View reports** for a list of DMARC reports by date.
-5. Select one of the dates shown to open a window with more details.
\ No newline at end of file
+5. Select one of the dates shown to open a window with more details.
+
+## Source details
+
+The Top 10 sources section shows you details about the top sources sending emails on your behalf, with information such as total volume of emails and how these sources fared regarding security policies.
+
+You also have access to information about all third parties, and can drill-down for further details on each of them:
```
Comment (pedrosousa) [inline content/dmarc-management/statistics.md]:
```suggestion
You also have access to information about all third parties, and can drill down for further details on each of them:
```

File: content/dmarc-management/statistics.md [hunk 2]
```diff
@@ -14,4 +14,16 @@ DMARC Management (beta) allows you to review how emails sent on your behalf have
...
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account and domain.
+2. Select **Email** > **DMARC Management**.
```
Comment (pedrosousa) [inline content/dmarc-management/statistics.md]: Nit: I would use "Go to" instead, but I see that you also used "Select" above. Do we have new guidelines for this navigational step?

**Review by pedrosousa — APPROVED** [review]

File: content/dmarc-management/statistics.md (hunk 2 above)
Comment (marciocloudflare) [reply content/dmarc-management/statistics.md]: Actually, no we don't Pedro. Just a fumble on my part. Will correct. Thanks

**Review by RZ7272 — COMMENTED** [review]
Run

**Pull request merged**